distributor-api-v1/deprecations/README.md

```diff
@@ -4,9 +4,10 @@ This page contains a list of past and future deprecations in the API. Deprecatio
 
 | Deprecation | Object | Property | Replaced by | Deprecation notice | Removal date |
 |:------------|:------------|:-------------------|:-------------|:-------------|:-------------|
-| [Average amount per night](./average-amount-per-night.md) | [Room price](../operations.md#room-price) | `AverageAmountPerNight` | `AverageAmountPerTimeUnit` | 21.7.2021 | 31.11.2021 |
-| [Maximum nights for settlement](./settlement-maximum-nights.md) | [Rate group](../operations.md#rate-group) | `SettlementMaximumNights` | `SettlementMaximumTimeUnits` | 21.7.2021 | 31.11.2021 |
-| [Product charging](./product-charging.md) | [Product](../operations.md#product) | `Charging` | `ChargingMode` | 21.7.2021 | 31.11.2021 |
-| [Product posting](./product-posting.md) | [Product](../operations.md#product) | `Posting` | `PostingMode` | 21.7.2021 | 31.11.2021 |
-| [Product pricing](./product-pricing.md) | [Product](../operations.md#product) | `Amounts` & `RelativePrice` | `Pricing` | 1.7.2021 | 31.11.2021 |
-| [Payment card input](./payment-card-input.md) | [Configuration](../operations.md#configuration) | `PaymentCardInput` | `PaymentCardRequirement` | 18.4.2021 | 31.7.2021 |
+| [Average amount per night](./average-amount-per-night.md) | [Room price](../operations.md#room-price) | `AverageAmountPerNight` | `AverageAmountPerTimeUnit` | 21.7.2021 | TBD |
+| [Maximum nights for settlement](./settlement-maximum-nights.md) | [Rate group](../operations.md#rate-group) | `SettlementMaximumNights` | `SettlementMaximumTimeUnits` | 21.7.2021 | TBD |
+| [Product charging](./product-charging.md) | [Product](../operations.md#product) | `Charging` | `ChargingMode` | 21.7.2021 | TBD |
+| [Product posting](./product-posting.md) | [Product](../operations.md#product) | `Posting` | `PostingMode` | 21.7.2021 | TBD |
+| [Product pricing](./product-pricing.md) | [Product](../operations.md#product) | `Amounts` & `RelativePrice` | `Pricing` | 1.7.2021 | TBD |
+| [Payment card input](./payment-card-input.md) | [Configuration](../operations.md#configuration) | `PaymentCardInput` | `PaymentCardRequirement` | 18.4.2021 | TBD |
+| [Category imageIds](./category-imageIds.md) | [Room category](../operations.md#room-category) | `ImageIds` | [Category image assignment](../operations.md#category-image-assignment) | 21.10.2021 | TBD |
```

Comment:
changed the `Removal date` to TBD since we need to also migrate and introduce a deprecation for `hotels/get` but there are still some bits missing.

Reply:
@koprivajakub : not wrong, just asking, is there an easy way I can check somewhere if the deprecation notice date is correct?

Reply:
Oh, I didn't read the description properly, it's clear to me now, it's explained there. No reason to answer this anymore.